arch/arm/core/cortex_m/cmse/arm_core_cmse.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <cortex_m/cmse.h>
+
+int arm_cmse_mpu_region_get(u32_t addr, u8_t *p_region)
+{
+	cmse_address_info_t addr_info =	cmse_TT((void *)addr);
+
+	if (addr_info.flags.mpu_region_valid) {
+		*p_region = addr_info.flags.mpu_region;
+	}
+
+	return addr_info.flags.mpu_region_valid;
+}
+
+static int _arm_cmse_addr_read_write_ok(u32_t addr, int force_npriv, int rw)
```

Comment:
the new standard is to prefix private kernel APIs with z_ and not _.
However, since this is a static function, the leading _ is not necessary.

Reply:
Agreed, remove leading underscore